fqdn.go

```diff
@@ -26,7 +26,7 @@ func Get() string {
 				return hostname
 			}
 			hosts, err := net.LookupAddr(string(ip))
-			if err != nil {
+			if len(hosts) == 0 || err != nil {
```

Comment:
Please, can you change the order of evaluation to `if err != nil || len(hosts) == 0`?

Thanks.